src/components/layout/navbar-items.tsx

```diff
@@ -19,34 +21,31 @@ export const NavbarItems: FC = () => {
         return '/'
     }
 
+    const currentDashboardUrl = dashboardURL()
```

Comment:
I know you didn't create this (it might even have been me?), but why are we creating a function and then calling it immediately?  also what does `current` mean in the variable name prefix? I don't see it changing, so maybe it's just `dashboardUrl`?

Instead it seems like we can just inline the code:
```suggestion
    let dashboardUrl = '/'
    if (isMember) dashboardUrl = '/member/openstax/dashboard'
    if (isResearcher) dashboardUrl = '/researcher/dashboard'
    if (isAdmin) dashboardUrl = '/admin/dashboard'
```

Reply:
good catch and I totally agree.